src/main.js

```diff
@@ -128,14 +128,15 @@ function renderResults(resultsList) {
             })' />\
         </label>`;
     });
-    getElementByClass('.iu_main-btn').remove();
```

Comment:
Any reason for doing `display: none;` instead of `.remove()` ?
`.remove()` removes the element from the DOM entirely,
and since there is no use for it later, I think it's slightly better here.